src/domains.js

```diff
@@ -62,7 +62,7 @@ export function getPayPalDomainRegex(): RegExp {
     return /.*loca.*/;
   }
   // eslint-disable-next-line security/detect-unsafe-regex
-  return /\.paypal\.(com|cn)(:\d+)?$/;
+  return /((\.paypal\.(com|cn))|(\.venmo\.com))(:\d+)?$/;
```

Comment:
I have a few suggestions/paths forward.

I think for this repo specifically, we should consider a `isVenmoDomain` function and a `isPayPalOwnedDomain` or `isPayPalTrustedDomain` function that combines `isPayPal` and `isVenmo`

If we start adding more domains like Honey, Braintree, or future products it'll make a lot of sense.

I'm guessing you are making this change for Zoid in `checkout-components`. I think we should really consider a new Zoid component for communicating with Venmo Web

Reply:
I like the idea of using a new method `isPayPalTrustedDomain()`. This will keep the method scalable to add more domains.